extensions/security/runtime/src/main/java/io/quarkus/security/runtime/QuarkusSecurityIdentity.java

```diff
@@ -205,6 +207,57 @@ public Builder addAttributes(Map<String, Object> attributes) {
             return this;
         }
 
+        /**
+         * Adds a permission.
+         *
+         * @param permission The permission
+         * @return This builder
+         */
+        public Builder addPermission(String permission) {
+            return addStringPermissions(Set.of(permission));
+        }
+
+        /**
+         * Adds permissions.
```

Comment:
Please mention it is a string permission. I know it is clear from the fact that it accepts a string or from implementation, but if you add `{@link StringPermission}` then IDE can show me without decompiling code (if I ere Quarkus user). It should take just a gist. Thanks